### PR TITLE
Nathan search

### LINKED ISSUE #15: Place notification system appropriately
Upon completion of Issue #14

### LINKED ISSUE #17: Search Component
Create a standalone component that allows for the manual search of users. 

Look at Figma for design specs. 
Interact with Redux Store for event name and attributes
Interact with Volunteer Api when updating text input

FOCUS ON MOBILE VIEW RATHER THAN WEB VIEW

### PR DESCRIPTION
**Pull request by OneWebDev**
closes #15 
closes #17

### CODE REVIEW

**Comment by AmanGotchu** [comment]
Do you have a lint error in Login.tsx?

Line 96:9 Expected an assignment or function call and isntead saw an expression

**Comment by OneWebDev** [comment]
fixed error

File: src/Pages/Search/Search.tsx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+      ).then(response => {
+        var responseData = response.data.results
+        
+        // OPTIONAL: Warning toast if there is 0 results found
+        responseData.length == 0 ? this.createWarningToast(responseData) : this.createSuccessToast(responseData) 
+      }).catch(error => {
+        this.createFailureToast() 
+      });
+    } else {
+      // Same error message as in the .catch tag
```
Comment (AmanGotchu) [inline src/Pages/Search/Search.tsx]: Redirect to Login anytime Token isn't available.

File: src/Components/toast.tsx [hunk 2]
```diff
@@ -41,9 +42,16 @@ class Toast extends React.PureComponent<IProps, IState> {
   generateToast = () => {
```
Comment (AmanGotchu) [inline src/Components/toast.tsx]: Is it possible to remove the loading line at the bottom of Toast? nit

**Pull request merged**